generator/src/main/java/com/scottlogic/deg/generator/walker/reductive/ReductiveState.java

```diff
@@ -12,7 +12,7 @@
 
     private final ProfileFields fields;
     private final Map<Field, FixedField> fixedFields;
-    private final FixedField lastFixedField;
+    public final FixedField lastFixedField;
```

Comment:
Only public for the implementation in `SystemOutDataGeneratorMonitor`. Should be changed to expose the name of the fixed field through an accessor method rather than breaking encapsulation.